nodejs/src/types.ts

```diff
@@ -65,6 +65,8 @@ export interface VoicesListEntryServer extends VoiceBasePropsServer {
 	// Voice avatar image URL.
 	avatar_image?: string | null;
 	gender?: "male" | "female" | "notSpecified";
+	preview_audio?: string | null;
```

Comment:
Is this also returned from voice creation? Previously the behavior was inconsistent, and voice creation would return fewer props than voice list. If it's still the case, then this change is all right, but if the inconsistency was fixed, we can simplify the interfaces here

Reply:
<img width="662" alt="Screenshot 2024-12-06 at 3 29 09 PM" src="https://github.com/user-attachments/assets/46d502ba-49f9-4c86-ad87-1f1714928f3f">
Yep, cloning still returns 4 fields only